src/SFA.DAS.Assessors.Api/Models/GetCourseListItem.cs

```diff
@@ -5,8 +5,12 @@ namespace SFA.DAS.Assessors.Api.Models
 {
     public class GetCourseListItem
     {
-        public int Id { get; set; }
+        public string StandardUId { get; set; }
+        public string IfateReferenceNumber { get; set; }
+        public int LarsCode { get; set; }
         public string Title { get; set; }
+        public decimal? Version { get; set; }
+        public string Status { get; set; }
         public int Level { get; set; }
         public int MaxFunding { get; set; }
```

Comment:
Do these need updating based on the updates by @ben1stone  ?

Reply:
yes... will have to wait till that is merged... I am too far from those changes :-(

Reply:
Yeah that's fair, it's hard to visualise